.gitignore

```diff
@@ -2,4 +2,6 @@ node_modules/
 .idea/
 .DS_Store
 .AppleDouble
-.LSOverride
\ No newline at end of file
+.LSOverride
+package-lock.json
```

Comment:
Why do we not want the lock?
AFAIK it's intended to exist in the git repo

Reply:
@HeverFarber you must have a lockfile or shrinkwrap to make the package consistently use the same dependency version on releases.

Reply:
But this is not true for NPM package. it's true for CI. each CI manage his package-lock and his deployment.

But when you install package from package.json (npm i <package>) - the main project only use "dependencies" and install them and added them to the main package-lock. 

so if we will manage package-lock in this repo it only affects us for CI. and our package-lock can be different from what the user will install. and we will publish bad version.

do you still think we need package-lock?

Reply:
@roni-frantchi @RLRabinowitz

Reply:
After we discuss. I will return it back For CI

Reply:
f4b0d52